Essay_Analysis_Tool/Controllers/MainViewController.cs

```diff
@@ -220,6 +221,11 @@ public void BuildAutocompleteMenu()
             _view.UpdateAutocompleteItems(items);
         }
 
+        public void Show()
+        {
+            _view.Show();
```

Comment:
When calling this function, I am getting a 'Parameter is not valid' Exception thrown with absolutely no information regarding where the exception was thrown from. I am not sure why this is occurring. Perhaps you could provide some insight on the issue?

Reply:
Try to Rebuild the Solution i didn't got that error but the MainView was just appeared for a second than it closed. After i changed the code to `_view.ShowDialog();` it started to work perfectly my wild guess is some Thread related caused this side-effect.

Reply:
Hey, big update here: I was reading about some different approaches to winforms applications, specifically applications that are mostly, if not all frontend dependent. There are actually design patterns and architectures that are intended for our use-case specifically. Basically what I am saying is what we have been trying to do by implementing the layered architecture here is quite silly since we don't have much of a backend. I am currently reading up on the command pattern and User Controls, which would provide a good way for us to divide up the functionality of the forms.

Reply:
These will more than likely be better for us by way of implementation and code navigation. If you have any thoughts on this I would love to hear them. I am honestly getting frustrated with the current look of the code as I am jumping back and forth between controller and view without having much of any model objects. This is because we aren't really making the text editor itself, but we are creating the tooling around the text editor that will allow users to interact with text editor. I really think that using either user controls or the command pattern (or maybe even both) could be our best bet as far as an application architecture for this application. What are your thoughts on this? Also I was thinking it would be a lot easier for us to hold a conversation over a different application, say discord or something, if you have a discord name or a preferred application please let me know!

Reply:
Yes, You are right the thin back-end and it sounds like a good plan to experiment with patterns. 
My Discord ID: *PoLáKoSz#4369*

Reply:
Looks like you'll actually have to add me, you have blocked friend requests: Zach Pedigo#7053